src/migrations/20171129064057_multicontainer.js

```diff
@@ -0,0 +1,304 @@
+
+var tryParse = function (obj) {
+	try {
+		return JSON.parse(obj)
+	}
+	catch(e) {
+		return {}
+	}
+}
+
+var singleToMulticontainerApp = function (app) {
+	// From *very* old supervisors, env or config may be null
+	// so we ignore errors parsing them
+	const conf = tryParse(app.config)
+	const env = tryParse(app.env)
+	const environment = {}
+	const appId = parseInt(app.appId)
+	for (let key in env) {
+		if (!/^RESIN_/.test(key)) {
+			environment[key] = env[key]
+		}
+	}
+	const newApp = {
+		appId: appId,
+		commit: app.commit,
+		name: app.name,
+		releaseId: 1,
+		networks: {},
+		volumes: {}
+	}
+	const defaultVolume = 'resin-data'
+	newApp.volumes[defaultVolume] = {}
+	let updateStrategy = conf['RESIN_SUPERVISOR_UPDATE_STRATEGY']
```

Comment:
Oh, you're right! For some reason I had myself convinced I couldn't require stuff here because of how webpack packages the migrations with the hack, but that's nonsense...